### ISSUE TITLE
Close button aria-label

### ISSUE THREAD

**Issue by swashbuck** [cgkineo/adapt-hotgrid#102]
### Subject of the issue

- In the popup, the aria-label for the Close button is missing

### Your environment
* Framework 5.24.8
* Hotgrid 4.3.2
